packages/frontend/src/components/connect-wallet/popovers/preferences/index.tsx

```diff
@@ -1,7 +1,12 @@
 import React, { useCallback } from "react";
```

Comment:
not related to this PR, but this file doesn't make sense inside of `connect-wallet` folder, as it's not related to the connected wallet.